tests/test_project.py

```diff
@@ -720,8 +708,8 @@ def test_JobsCursor_contains(self):
 
     def test_job_move(self):
         root = self._tmp_dir.name
-        project_a = signac.init_project("ProjectA", os.path.join(root, "a"))
-        project_b = signac.init_project("ProjectB", os.path.join(root, "b"))
+        project_a = signac.init_project(os.path.join(root, "a"))
+        project_b = signac.init_project(os.path.join(root, "b"))
```

Comment:
I would recommend to keep at least a few tests that test the legacy behavior until we have made a decision on how to deal with backwards-compatibility for this particular function. It is too widely used to just break. See discussion above.

Reply:
I changed all the tests to make sure that I didn't miss anything in my first round of changes because I wanted reviewers to be able to see the full scope of the change. I won't bother modifying any more until we've decided on the correct semantics.